### ISSUE TITLE
Set agent heartbeat rate via exchange node resource

### ISSUE THREAD

**Issue by dabooz** [open-horizon/anax#1310]
Currently, the agent heartbeats to the exchange when it asks the exchange for any changes since the last heartbeat. The heartbeat rate varies from once every 10 sec to once every 2 mins, with an adjustment of 10 secs max. Those values are not easily configurable. Add fields to the exchange which will hold these values. Use it to override whatever is in the agent config file.

The exchange will have to be updated before you can work on this.

**Comment by bmpotter** [comment]
The field i'm adding to the Node resource is:

```
"heartbeatIntervals": {
  "minInterval": 10,
  "maxInterval": 120,
  "intervalAdjustment": 10
}
```

It is optional, if it is not specified all fields will be set to `0` which means use the default value in the anax config file in the agent.

**Comment by bmpotter** [comment]
The exchange work above has been shipped in exchange version 2.1.0

**Comment by dabooz** [comment]
This API change was exploited within the PR for #1311
